package.json

```diff
@@ -59,7 +59,7 @@
     "d3"
   ],
   "bugs": {
-    "url": "https://github.com/tomshanley/d3-sankey-circular/issues"
+    "url": "https://github.com/plotly/d3-sankey-circular/issues"
```

Comment:
It's probably fine for bugs to be reported upstream. Maybe we should keep the original URL here.